cypress/e2e/api/testsApi.cy.js

```diff
@@ -0,0 +1,162 @@
+describe('Validate the API requests', () => {
+  const CUSTOMERS_API_URL = `${Cypress.env('API_URL')}/customers`
+  context('Postive scenarios', () => {
+    context('Pagination and limits', () => {
+      it('Get customers successfully', () => {
+        cy.request({
+          method: 'GET',
+          url: `${CUSTOMERS_API_URL}?page=1&limit=10`,
+        }).then(response => {
+          expect(response.status).to.equal(200)
+        })
+      })
+
+      it('Change the page and check the first customers', () => {
+        let firstCustomerId = '';
+        cy.request({
+          method: 'GET',
+          url: `${CUSTOMERS_API_URL}?page=1&limit=10`,
+        }).then(response => {
+
+          expect(response.status).to.equal(200)
+          firstCustomerId = response.body.customers[0].id;
+        })
+
+        cy.request({
+          method: 'GET',
+          url: `${CUSTOMERS_API_URL}?page=2&limit=10`,
+        }).then(response => {
+
+          expect(response.status).to.equal(200);
+          expect(response.body.customers[0].id).to.not.equal(firstCustomerId)
+        })
+      })
+
+      it('Change the limits of customers per page', () => {
+        cy.request({
+          method: 'GET',
+          url: `${CUSTOMERS_API_URL}?page=1&limit=10`,
+        }).then(response => {
+
+          expect(response.status).to.equal(200);
+          expect(response.body.pageInfo.totalPages).to.equal(5);
+        })
+
+        cy.request({
+          method: 'GET',
+          url: `${CUSTOMERS_API_URL}?page=1&limit=20`,
+        }).then(response => {
+
+          expect(response.status).to.equal(200);
+          expect(response.body.pageInfo.totalPages).to.equal(3);
+        })
+      })
+    });
+    context('Filters', () => {
+      it('Filter small size companies', () => {
+        cy.request({
+          method: 'GET',
+          url: `${CUSTOMERS_API_URL}?page=1&limit=10&size=Small&industry=All`,
```

Comment:
Visto que o teste se propões em testar filtragem por _size_, passe apenas tal _query string_ à API.

Para as outras deixe o padrão.

Ou seja, aqui poderia ser assim:

```suggestion
          url: `${CUSTOMERS_API_URL}?size=Small`,
```

Reply:
O mesmo vale para os próximos testes.

Evite complexidade quando as coisas podem ser simples.